modules/mobile/app/serializers/mobile/v0/prescription_tracking_serializer.rb

```diff
@@ -6,9 +6,11 @@ class PrescriptionTrackingSerializer
       include FastJsonapi::ObjectSerializer
 
       set_type :PrescriptionTracking
-      set_id(&:prescription_id)
+      set_id :tracking_number
```

Comment:
setting tracking number as id since it is the only unique number between all the objects in the array. I also kept the tracking number field to not add any confusion.